docs/getting_started.md

```diff
@@ -15,7 +15,7 @@ At the moment, Thanos is able to use [different storage providers](storage.md),
 
 You can find the latest Thanos release [here](https://github.com/improbable-eng/thanos/releases).
 
-If you want to build Thanos from source -
+If you want to build Thanos from source, make sure you have installed `bzr` and `git`(check with `which bzr` and `which git`) -
```

Comment:
It seems like a space is missing before `(`. Plus, the `-` looks off here. Perhaps let's say that `, and that you have a working installation ...`?

Reply:
the `-` is the old thing, since useless, i will remove it.